packages/strapi-plugin-content-type-builder/admin/src/containers/Form/sagas.js

```diff
@@ -29,21 +29,17 @@ export function* editContentType() {
       method: 'PUT',
       body,
     };
+    const shouldWatchServerRestart = true;
```

Comment:
Why are you declaring this variable if the value is always set to true?

Reply:
It's for information purpose